internal/pkg/describe/env_test.go

```diff
@@ -152,3 +152,126 @@ func TestEnvDescriber_Describe(t *testing.T) {
 		})
 	}
 }
+
+func TestEnvDescription_JSONString(t *testing.T) {
+	testProject := &archer.Project{
+		Name: "testProject",
+		Tags: map[string]string{"key1": "value1", "key2": "value2"},
+	}
+	testEnv := &archer.Environment{
+		Project:          "testProject",
+		Name:             "testEnv",
+		Region:           "us-west-2",
+		AccountID:        "123456789012",
+		Prod:             false,
+		RegistryURL:      "",
+		ExecutionRoleARN: "",
+		ManagerRoleARN:   "",
+	}
+	testApp1 := &archer.Application{
+		Project: "testProject",
+		Name:    "testApp1",
+		Type:    "load-balanced",
+	}
+	testApp2 := &archer.Application{
+		Project: "testProject",
+		Name:    "testApp2",
+		Type:    "load-balanced",
+	}
+	testApp3 := &archer.Application{
+		Project: "testProject",
+		Name:    "testApp3",
+		Type:    "load-balanced",
+	}
+	allApps := []*archer.Application{testApp1, testApp2, testApp3}
+	wantedContent := "{\"environment\":{\"project\":\"testProject\",\"name\":\"testEnv\",\"region\":\"us-west-2\",\"accountID\":\"123456789012\",\"prod\":false,\"registryURL\":\"\",\"executionRoleARN\":\"\",\"managerRoleARN\":\"\"},\"applications\":[{\"project\":\"testProject\",\"name\":\"testApp1\",\"type\":\"load-balanced\"},{\"project\":\"testProject\",\"name\":\"testApp2\",\"type\":\"load-balanced\"},{\"project\":\"testProject\",\"name\":\"testApp3\",\"type\":\"load-balanced\"}],\"tags\":{\"key1\":\"value1\",\"key2\":\"value2\"}}\n"
+
+	t.Run(wantedContent, func(t *testing.T) {
```

Comment:
we don't need to run `t.Run` :) also putting the wantedContent in [t.Run](https://golang.org/pkg/testing/#T.Run) is a bit odd, since the first parameter is supposed to be the name of a subtest